File: Lab1/lab1.java
public class lab1 {

    public static int[] findSum(int arr[], int target)
    {
        int sum = -1;
        int index [] = new int[2];
        for(int i = 0; i < arr.length; i++)
        {
            for(int j = i+1; j < arr.length; j++)
            {
                sum = arr[j] + arr[i];
                if(sum == target)
                {
                    index[0] = j;
                    index[1] = i;
                    return index;
                }
            }
        }
        return null;
    }
    public static void main(String[] args)
    {
        int arr [] = {2, 3, 45, 2, 6, 7, 8};
        int target = 4;
        int [] index = findSum(arr, target); 
        System.out.println("indexes are: " + index[0] + ", " + index[1]);
    }
}

File: Lab1/Lab1.py
def find_sum(arr, target):
    index = [-1, -1]
    for i in range(len(arr)):
        for j in range(i + 1, len(arr)):
            if arr[i] + arr[j] == target:
                index[0] = j
                index[1] = i
                return index
    return None

def main():
    arr = [2, 3, 45, 2, 6, 7, 8]
    target = 4
    index = find_sum(arr, target)
    if index is not None:
        print("indexes are:", index[0], ",", index[1])
    else:
        print("No such pair found")

if __name__ == "__main__":
    main()
